explainer.md

```diff
@@ -99,19 +99,20 @@ async function OnVRAvailable() {
   // has that capability the page will want to add an "Enter VR" button (similar
   // to "Enter Fullscreen") that triggers the page to begin showing imagery on
   // the headset.
-  let exclusiveMode = await vrDevice.supportsSession({ exclusive: true });
-  if (exclusiveMode) {
+  vrDevice.supportsSession({ exclusive: true }).then(() => {
     var enterVrBtn = document.createElement("button");
     enterVrBtn.innerHTML = "Enter VR";
     enterVrBtn.addEventListener("click", BeginVRSession);
     document.body.appendChild(enterVrBtn);
-  }
+  }).catch(() => {
+    console.log("VRDevice does not support exclusive sessions.");
```

Comment:
Good call. I've updated the example code to log out a "reason" argument provided by the promise rejection.